test/unit-test/test_commands_bootstrap.py

```diff
@@ -64,7 +64,9 @@ def session_manager(sts_client):
 def test_deploy_template(session_manager, mocker):
     mocker.patch("seedfarmer.commands._bootstrap_commands.role_deploy_status",return_value={"RoleName":"BLAHBLAH"})
     mocker.patch("seedfarmer.commands._bootstrap_commands.cs_services.cfn.deploy_template",return_value=None)
-    template = bc.get_toolchain_template(project_name="myapp",principal_arn=['arn:aws:iam::123456789012:role/AdminRole']
+    template = bc.get_toolchain_template(project_name="myapp",
+                                         role_name="seedfarmer-test-toolchain-role",
+                                         principal_arn=['arn:aws:iam::123456789012:role/AdminRole']
```

Comment:
this formatting looks off.

Reply:
black wouldn't format it aligned to the `(`.

Reply:
hmmm....looking into it

Reply:
Formatted all tests ;).